lib/modules/manager/terraform/modules.ts

```diff
@@ -64,7 +64,13 @@ export function analyseTerraformModule(dep: PackageDependency): void {
     dep.datasource = BitBucketTagsDatasource.id;
   } else if (gitTagsRefMatch?.groups) {
     dep.depType = 'module';
-    if (gitTagsRefMatch.groups.path.includes('//')) {
+    dep.currentValue = gitTagsRefMatch.groups.tag;
+    dep.datasource = GitTagsDatasource.id;
+    if (azureDevOpsSshRefMatch?.groups) {
```

Comment:
shouldn't we lift this before normal tags match? like we do for GitHub and bitbucket ...

Reply:
I think it doesn't matter because now we will set `currentValue` to `gitTagsRefMatch.groups.tag` and if the URL is `azure` URL we will change the value.

Reply:
sure, but code seems to look cleaner and less nested then

Reply:
sure, so do you want me to revert the last changes? to return these two lines above after normal tags?

Reply:
yes, i think best place it between bitbucket tags and git tags conditions.